tests/integration/utils.go

```diff
@@ -15,8 +15,9 @@ import (
 )
 
 type RequestTestCase struct {
```

Comment:
todo: Please revert this file - we should not be extending this old legacy type that we are trying to remove.